imports/plugins/core/router/lib/router.js

```diff
@@ -650,6 +650,16 @@ Router.initPackageRoutes = (options) => {
         }
       });
 
+      // // Static Pages View route
```

Comment:
Why is this commented? if it's not serving any purpose then it shouldn't be here.